src/features/post/new/PostEditorRoot.tsx

```diff
@@ -77,6 +80,8 @@ const HiddenInput = styled.input`
 
 type PostType = "photo" | "link" | "text";
 
+const MAX_TITLE_LENGTH = 200;
```

Comment:
@aeharding is this a limit imposed by the back end? :thinking:

Reply:
Yeah - it's hardcoded in Lemmy last I checked (https://github.com/iav/lemmy/commit/65a6e7731c632313f539d232dec4f6f6c8a49084)